lib/braintree/subscription.js

```diff
@@ -35,7 +35,18 @@ class Subscription extends AttributeSetter {
 
   constructor(attributes) {
     super(attributes);
-    this.transactions = attributes.transactions.map((transactionAttributes) => new Transaction(transactionAttributes));
+
+    if (!attributes.transactions) {
```

Comment:
I'm wary of doing this to accommodate what is a problem with the `js2xmlparser` library. Is there another solution that doesn't involve making a change to this file to work around the limitation in the package?